packages/ia-components/sandbox/bookreader-component/bookreader-wrapper-main.jsx

```diff
@@ -38,7 +39,6 @@ export default class BookReaderWrapper extends Component {
     const originalGetPageURI = BookReader.prototype.getPageURI;
     const defaultOptions = {
       el: `#${this.BookReaderRef.current.id}`,
-      mobileNavFullscreenOnly: true,
```

Comment:
QUESTION: Why was this removed?

Reply:
thanks for catching. i've re-added.
did not mean to remove until we move into item-navigator world